pycaching/geocaching.py

```diff
@@ -584,4 +587,8 @@ def load_trackable_list(self, url):
         trackable_table = root.find_all("table")[1]
         links_raw = trackable_table.find_all("a")
         urls = [l.get("href") for l in links_raw if "track" in l.get("href")]
-        return [self.load_trackable_by_url(t) for t in urls]
+        names = [re.split("[\<\>]", str(l))[2] for l in links_raw if "track" in l.get("href")]
```

Comment:
Could you please rename variables / add a comment / do somehow this line more readable?